apps/system/test/marionette/notification_actions_test.js

```diff
@@ -58,4 +68,25 @@ marionette('launch an app via notification click', function() {
     });
   });
 
+  test('swiping to dismiss', function() {
```

Comment:
While I don't really have much of an opinion either way, I thought we were trending towards one test per file for marionette tests?